test/jdk/sun/security/provider/named/NamedEdDSA.java

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
I didn't expect an EdDSA test in this PQC bug fix.

Reply:
This is an existing test. It just demonstrates how the `NamedXyz` classes can be used to implement any algorithm that can be configured with a parameter set name.